README.md

```diff
@@ -61,6 +61,7 @@ Gazebo Classic and Ignition software.
         1. Classic release: `<path to release-tools>/release.py -r 1 gazebo11 11.2.0 secrettoken --dry-run`
     1. If the dry run succeeds, run the same command again, now without `--dry-run`.
 1. Check that:
-    * Several `-debbuilder` jobs have been queued in https://build.osrfoundation.org/
+    * Several `-debbuilder` jobs have been queued in https://build.osrfoundation.org/ and watch those jobs to see if any of them fail.
```

Comment:
Should we mention things users can do if any of the jobs fail?

Reply:
I'm thinking about this and trying to figure out what those things are

Reply:
I've added some additional steps to take in 6bc264e. Please take a look

Reply:
This looks good - the only thing I would consider adding is a link to http://packages.osrfoundation.org/gazebo/ - we can mention that users can see previous releases here if they're troubleshooting failed builds and want to see if a particular build has failed in the past

Reply:
how does ad48124 look?

Reply:
Looks great!